src/gui/screen_menu_statistics.cpp

```diff
@@ -4,11 +4,14 @@
 #include "screen_menu_statistics.hpp"
 #include "DialogMoveZ.hpp"
 #include "img_resources.hpp"
+#include "app_metrics.h"
 
 ScreenMenuStatistics::ScreenMenuStatistics()
     : ScreenMenuStatistics__(_(label)) {
     EnableLongHoldScreenAction();
     header.SetIcon(&img::info_16x16);
+    AddItem(&uptime_label);
```

Comment:
This is not a valid code for adding menu items in the menu.

1. The menu must be added to the `ScreenMenu` template as a parameter
2. AddItem function is nowhere in the firmware
3. Use `MenuItemAutoUpdatingLabel` for an automatically updating label